recipes-devtools/python/files/cross-compiling.patch

```diff
@@ -0,0 +1,22 @@
+Index: pycrypto-2.6/setup.py
```

Comment:
Should we put `Upstream-Status` into this patch?

Reply:
I was thinking about this as well; but then deferred the activity for a next pull request because I would need to understand the whole rationale of the patch, re-build without the patch, see the error message, understand the patch and look if it has been contributed somewhere. Nevertheless, it should be added and should be really added to meta-openembedded and meta-ros. I just did not do it here for this pull request yet.